app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] 🚀 Outstanding work in establishing the associations between your models! 👏 However, there are a few changes that can further enhance your codebase and bring it closer to achieving the desired results.

   - The `belongs_to` association in the post model should reference a specific column on the user table. This column is specified as the `foreign_key`. In our case, this column is 'author'. Additionally, it's beneficial to provide a `class_name` that points to the specific table, which in our case is the 'User' table.

   Instead of:
   ```ruby
   belongs_to :user
   ```

   I recommend:

   ```ruby
   belongs_to :author, class_name: 'User'
   ```

By making these adjustments, your code will not only be more precise but also better aligned with best practices. 
Happy coding! 😊👍